polaris/hub/client.py

```diff
@@ -432,7 +432,7 @@ def upload_dataset(
         self,
         dataset: Dataset,
         access: AccessType = "private",
-        timeout: TimeoutTypes = (10, 200),
+        timeout: TimeoutTypes = TimeoutTypes(connect=10, write=200),
```

Comment:
I don't think it works that way - looks like you need to either use a `httpx.Timeout` object or stick with a tuple and simply explain what the tuple is about in the docstring.

Reply:
I think a tuple is best (one less import for our users)

Reply:
Yeah, my bad! Should be good now!

Reply:
Thanks! LGTM for me.